forge-gui/res/cardsfolder/upcoming/the_fall_of_kroog.txt

```diff
@@ -0,0 +1,10 @@
+Name:The Fall of Kroog
+ManaCost:4 R R
+Types:Sorcery
+A:SP$ ChoosePlayer | ValidTgts$ Player.Opponent | TgtPrompt$ Select target opponent | Choices$ TargetedPlayer | SubAbility$ DBDestroy | StackDescription$ SpellDescription | RememberOriginalTargets$ True | SpellDescription$ Choose target opponent. Destroy target land that player controls. CARDNAME deals 3 damage to that player and 1 damage to each creature they control.
+SVar:DBDestroy:DB$ Destroy | ValidTgts$ Land.ControlledBy ParentTarget,Land.ControlledBy Remembered | TgtPrompt$ Select target land that player controls | SubAbility$ DBDealDamage
+SVar:DBDealDamage:DB$ DealDamage | Defined$ TargetedPlayer | NumDmg$ 3 | SubAbility$ DBDamageAll | DamageMap$ True
```

Comment:
Does this need a StackDescription$ None ? You will be able to tell by looking at the StackDescription when cast which lines are making auto stack descs